test/kernel_example.f90

```diff
@@ -242,12 +292,138 @@ program xc_example
 !  second derivatives it might be better to compute all partial derivatives and
 !  reuse them in the response solver.
 !
-!     In this example we use the variables that are linear in the density matrix.
+!  In this example we use the variables that are linear in the density matrix.
 !  This makes life easier because you can construct them trivially from perturbed
 !  density matrices. If you use i.e. the square norm of the density gradient
 !  things get more complicated.
 !
-!     For the spin polarized cases (using density and spin density or up- and down
+!  For the spin polarized cases (using density and spin density or up- and down
 !  densities) we would use 8 variables instead of 4.
+!
+!  This will be exemplified in the following.
+
+  
+  
+  ! Now contract the 2nd derivative completely
+  !  allocate density and fill in some phantasy values
```

Comment:
Nitpick: "phantasy" --> "fantasy"

Reply:
You can fix this by committing a change to the same branch that you have used and pushing the same branch to your fork. This will append to this pull request.